WalletWasabi.Packager/NSubsys/PeUtility.cs

```diff
@@ -67,22 +68,20 @@ public void Dispose()
 			InternalBinReader?.Dispose();
 		}
 
-#pragma warning disable IDE1006 // Naming Styles
-
 		[StructLayout(LayoutKind.Explicit)]
 		public struct ImageDosHeader
 		{
 			[FieldOffset(60)]
+			[SuppressMessage("Style", "IDE1006:Naming Styles", Justification = "")]
```

Comment:
No justification here?

Reply:
The reason we have this naming violation is because we are using a public field (in editorconfig we have a rule to not use public fields https://github.com/zkSNACKs/WalletWasabi/blob/master/.editorconfig#L164-L174)
I initially thought that these fields must be public but maybe they can be internal. https://github.com/zkSNACKs/WalletWasabi/pull/4406#issuecomment-696153574

Reply:
I removed that attribute.